src/index.scss

```diff
@@ -0,0 +1,12 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  height: 100vh;
```

Comment:
Perfect remove the `height: 100vh;` from the PageLayout

Reply:
Fixed